hexane/atomistic/run.sh

```diff
@@ -1,11 +1,17 @@
 #! /bin/bash -e
 
-grompp -v
+gmx grompp -v
 
-mdrun -v
+gmx mdrun -v
+
+echo Running Extract_Energies.sh to extract all thermodynamic quantities from ener.edr
+#equilibration time in Gromacs units (ps)
+equi=200
+echo equi = $equi
+./Extract_Energies.sh $equi
 
 echo Calculating distributions
-csg_stat --top topol.tpr --trj traj.trr --cg hexane.xml --nt 3 --options fmatch.xml
+csg_stat --top topol.tpr --trj traj.trr --cg hexane.xml --nt 3 --options settings.xml --begin $equi
```

Comment:
Instead of hard coding the number of threads, do:
```
nt="$(grep -c processor /proc/cpuinfo 2>/dev/null)" || nt=0
((nt++))
```
(logic from `build.sh`)